sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -47,364 +48,444 @@ const alwaysBeDeletedSubscription = "alwaysbedeletedsubscription";
-          case EntityType.QUEUE:
-          case EntityType.TOPIC:
-            try {
-              response = await deleteEntity(entityType, "notexisting");
-            } catch (err) {
-              response = err;
-            }
-            break;
-
-          case EntityType.SUBSCRIPTION:
-            try {
-              response = await deleteEntity(entityType, "notexisting", alwaysBeExistingTopic);
-            } catch (err) {
-              response = err;
-            }
-            break;
-
-          case EntityType.RULE:
-            try {
-              response = await deleteEntity(
-                entityType,
-                "notexisting",
-                alwaysBeExistingTopic,
-                alwaysBeExistingSubscription
-              );
-            } catch (err) {
-              response = err;
-            }
-            break;
-
-          default:
-            throw new Error("TestError: Unrecognized EntityType");
-        }
-        should.equal(response.statusCode, 404);
-      });
+    it(`Lists on existing entities for type ${testCase.entityType} with skip 1 returns entities skipping first one`, async () => {
+      let response;
 
-      it(`Deletes an existent ${entityType} entity successfully`, async () => {
-        let response;
-        switch (entityType) {
-          case EntityType.QUEUE:
-            try {
-              await createEntity(entityType, alwaysBeDeletedQueue);
-            } catch (err) {
-              console.log("Ignoring clean up step");
-            }
-            response = await deleteEntity(entityType, alwaysBeDeletedQueue);
-            break;
-
-          case EntityType.TOPIC:
-            try {
-              await createEntity(entityType, alwaysBeDeletedTopic);
-            } catch (err) {
-              console.log("Ignoring clean up step");
-            }
-            response = await deleteEntity(entityType, alwaysBeDeletedTopic);
-            break;
-
-          case EntityType.SUBSCRIPTION:
-            try {
-              await createEntity(entityType, alwaysBeDeletedSubscription, alwaysBeExistingTopic);
-            } catch (err) {
-              console.log("Ignoring clean up step");
-            }
-            response = await deleteEntity(
-              entityType,
-              alwaysBeDeletedSubscription,
-              alwaysBeExistingTopic
-            );
-            break;
-
-          case EntityType.RULE:
-            try {
-              await createEntity(
-                entityType,
-                alwaysBeDeletedRule,
-                alwaysBeExistingTopic,
-                alwaysBeExistingSubscription
-              );
-            } catch (err) {
-              console.log("Ignoring clean up step");
-            }
-            response = await deleteEntity(
-              entityType,
-              alwaysBeDeletedRule,
-              alwaysBeExistingTopic,
-              alwaysBeExistingSubscription
-            );
-            break;
+      try {
```

Comment:
Same comment about the try/catch as above. Would be helpful to remove to reduce false positives/make failures in CI easier to troubleshoot.

Reply:
Updated, also see https://github.com/Azure/azure-sdk-for-js/pull/6112#issuecomment-555716170